neo4j/internal/bolt/bolt3.go

```diff
@@ -617,8 +617,8 @@ func (b *bolt3) receiveNext(ctx context.Context) (*db.Record, *db.Summary, error
 	}
 }
 
-func (b *bolt3) Bookmark() string {
-	return b.bookmark
+func (b *bolt3) Bookmark() (string, string) {
+	return b.bookmark, ""
 }
```

Comment:
We may need some docs here. I took some time to figure out the return from this method.

Reply:
It's documented at the interface level: https://github.com/neo4j/neo4j-go-driver/blob/9cd76fe4dc146d92df981c2bae0eb5e20628a5e0/neo4j/internal/db/connection.go#L82-L87
Do you think I should add a specific comment for that empty string here? Or sth else?

Reply:
It may worth it since we were not that close to the context where the interface is defined.

Reply:
77d53e34897bf4e9c3ab754a60e59f385935d6e8